deny.toml

```diff
@@ -70,26 +70,24 @@ skip-tree = [
 
     # wait for zcashd and zcash_script to upgrade
     # https://github.com/ZcashFoundation/zcash_script/pulls
-    { name = "metrics", version = "=0.20.1" },
     { name = "sha2", version = "=0.9.9" },
```

Comment:
The only remaining dependency on this older sha2 version is in zebra-chain, so we can upgrade it in this PR and remove this exception. This is part of #6638.